app/sse.go

```diff
@@ -0,0 +1,106 @@
+// Copyright © 2022-2025 Obol Labs Inc. Licensed under the terms of a Business Source License 1.1
+
+package app
+
+import (
+	"context"
+	"encoding/json"
+	"strconv"
+
+	"github.com/obolnetwork/charon/app/errors"
+	"github.com/obolnetwork/charon/app/log"
+	"github.com/obolnetwork/charon/app/sseclient"
+	"github.com/obolnetwork/charon/app/z"
+	"github.com/obolnetwork/charon/eth2util"
+)
+
+const (
+	sseHead       = "head"
+	sseChainReorg = "chain_reorg"
+)
+
+type SSEHead struct {
+	Slot                      string `json:"slot"`
```

Comment:
Can this be uint64 to avoid str->int conversion after json.Unmarshal?

Reply:
By the spec we are receiving a string...

Reply:
OMG